Documentation/PageTranslations/Index.rst

```diff
@@ -6,10 +6,10 @@
 Translating pages
 =================
 
-When you want to translate a page in the page tree you create an
-"alternative page language" record on that page. This contains fields
-similar to that of the page record of the default language which you fill in
-with translated content.
+In order to translate a page in the page tree you create a
+copy of that record starting by its parent page. 
```

Comment:
```suggestion
copy of that page with a new language relation by using the module :guilabel:`Page > Language Comparison` and the dropdown :guilabel:`Create a new translation of this page`. 
```

I think we should then prevent the use of the term "record", and only refer to "page"?


And I think we need to describe the GUI method more properly? Otherwise if it just says "create a copy" people think they really should copy a page, and not using the proper "Create a new translation of this page" way.

Reply:
I think it would be helpful to mention here that the "Language Comparison" is a select box on the top of the right page. For me it has been hard to figure this out.

Reply:
> I think it would be helpful to mention here that the "Language Comparison" is a select box on the top of the right page. For me it has been hard to figure this out. And it should be mentioned how the Select Box works. You need to choose the language and then it starts immediately.

I agree, do you think my proposed suggestion doesn't address this? Feel free to make an alternative wording suggestion.